steps/incident-update/step.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "endpoint_url": {
```

Comment:
In general (for all integrations), these should be `camelCase` and not `snake_case`. Also, do we need a new connection type for VictorOps? Would that make these feel nicer?

Reply:
the connection would be good eventually but there's only 1 field that's necessary (a URL with an obfuscated token embedded in it) so it's not a big impediment